README.md

```diff
@@ -87,7 +87,7 @@ server.start();
 or extend the provided HTTP server and override any methods you need:
 ```js
 // my-custom-express-server.js
-const ExpressHTTPServer = require('fastboot-app-server/lib/express-http-server');
+const ExpressHTTPServer = require('fastboot-app-server/src/express-http-server');
```

Comment:
Like https://github.com/ember-fastboot/fastboot-app-server/pull/40/files, I think this should point to `fastboot-app-server/dist/cjs/express-http-server`

@tomdale Can you confirm?